plugins/TimingTriggerCandidateMaker.hpp

```diff
@@ -55,6 +55,17 @@ class TimingTriggerCandidateMaker : public dunedaq::appfwk::DAQModule
 
   std::string m_hsievent_receive_connection;
 
+  // HSI Passthrough changes
+  std::atomic<bool> m_hsi_passthrough;
+  int hsi_pt_before;
```

Comment:
DAQ coding style is to prefix member variables in classes with `m_`, so `m_hsi_pt_before`, `m_hsi_pt_after`, `m_trigger_bitmask` and `m_low_high_bits`

Reply:
Style guide is here: https://github.com/DUNE-DAQ/styleguide

Reply:
understood, renamed.